src/query/__tests__/buildRQL-test.js

```diff
@@ -36,8 +36,11 @@ describe('buildRQL', () => {
     });
     MockComponent = React.createClass({render});
     MockContainer = Relay.createContainer(MockComponent, {
+      initialVariables: {
+        if: null,
+      },
       fragments: {
-        foo: () => Relay.QL`fragment on Node { name }`,
+        foo: () => Relay.QL`fragment on Node { firstName(if: $if) }`,
```

Comment:
Rather than to edit `MockContainer` could you push your custom mock down into your test? See `MockContainer2`.